<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_judul">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Impact&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>IMPLEMENTASI ALGORITMA CNN UNTUK KLASIFIKASI KUALITAS BUAH APEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>631</width>
      <height>612</height>
     </rect>
    </property>
    <property name="title">
     <string>Proses Pelatihan</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>41</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kualitas Buah</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboKualitasBuah">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>59</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 255, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apel bagus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apel busuk</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnBacaCitraLatih">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>87</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string>Baca Data Latih</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProcessing">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>141</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cropping</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnfolderkos">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>6</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string>Buat Folder Kosong</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>123</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Preprocessing</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>256</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Total Data latih</string>
     </property>
     <widget class="QLineEdit" name="editTotalDataLatih">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="btnProsesPelatihan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Proses Pelatihan</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Epoch</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editIterasi">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>7</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Akurasi Pelatihan</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>11</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Training Akurasi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Validation Akurasi</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ediTrainingAkurasi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>32</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="editValidationAkurasi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Training Loss </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>132</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Validation Loss</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editTrainingLoss">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="editValidationLoss">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>147</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>154</x>
       <y>7</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Latih</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tblDataLatih">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>24</y>
       <width>461</width>
       <height>261</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <column>
      <property name="text">
       <string>Nama Citra</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kualitas Buah</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kode Kualitas Buah</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="lblGrafikKonversi">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>451</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Grafik </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblGrafikKonversi_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>450</y>
       <width>451</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Grafik </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
